projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.ts

```diff
@@ -137,12 +145,12 @@ export class GoCalendarComponent implements OnDestroy, OnInit, AfterViewChecked
     if (this.appendTo === 'body') {
       const calenderBodyPosition: any = this.calendarView.nativeElement.getBoundingClientRect();
       Object.assign(this.calendarView.nativeElement.style, {
-        height: `${calenderBodyPosition.height}px`,
-        bottom: `${calenderBodyPosition.bottom}px`,
-        top: `${calenderBodyPosition.top}px`,
+        bottom: `${calenderBodyPosition.bottom - 5}px`,
         right: `${calenderBodyPosition.right}px`,
         width: `${calenderBodyPosition.width}px`,
         left: `${calenderBodyPosition.left}px`,
+        height: this.displayAbove ? 'auto' : `max-content`,
```

Comment:
`max-content` is not supported by IE11 and has partial support in Firefox and Safari. https://caniuse.com/?search=max-content

I am curious why we need to change any of the positioning of the datepicker? It does not seem related to the original issue that this PR is addressing and the current way of positioning the datepicker seems to work just fine as far as I can see.

Reply:
Yes. You are right but when I was working that time I was facing some another issue like when I was changing the month so that particular Row also changing (Like **Feb** is 28 days and **March** has 31 days ), So Row will increase or decrease but Auto Hight was not working particular Calendar body.  So for that reason I had update the css but anyway I will call you tomorrow and show you what is the exact issue.